tests/debug.rs

```diff
@@ -1820,6 +1820,81 @@ mod generic {
         }
     }
 
+    mod raw {
+        #[cfg(not(feature = "std"))]
+        use alloc::format;
+
+        use derive_more::Debug;
+
+        #[derive(Debug)]
+        struct StructOne<T> {
+            r#thing: T,
```

Comment:
I think we should also have a test for the case where the `r#` is actually needed, like for a keyword such as `r#struct`